lib/plugins/features/movies/index.js

```diff
@@ -19,7 +20,27 @@ exports.register = (server, options, next) => {
 
   next();
 };
+exports.find = (server, options, next) => {
+  server.route([{
```

Comment:
This route should go within the list provided to the `server.route` call in `exports.register`, e.g. 

```js
server.route([
  {
    method: 'POST',
    ...
  },
  {
    method: 'GET'
  }
];
```

That way you don't have to have a separate `find` function, and you don't need to define `exports.find.attributes` on line 43.